runelite-api/src/main/java/net/runelite/api/Constants.java

```diff
@@ -67,4 +67,8 @@
 	 * the plane value to 0-3.
 	 */
 	public static final int MAX_Z = 4;
+	/**
+	 * The number of milliseconds in a client tick.
```

Comment:
Might be worth mentioning that this is the length of one frame when the client is running at its max framerate of 50fps.